db/main.go

```diff
@@ -33,6 +34,15 @@ func NewSpdTenantSvc() *SpdTenantSvc {
 	return &SpdTenantSvc{DB: dbi, DryRunMode: config.DRY_RUN, Config: config}
 }
 
+func SpdTenantSvcMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
```

Comment:
what's this function for?

Reply:
Was left over from other changes will remove